website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -9,6 +9,9 @@ description: >-
 
 Keep track of changes to the API for HCP Terraform and Terraform Enterprise.
 
+## 2024-10-14
+* Update the [Organizations API](/terraform/cloud-docs/api-docs/organizations) to support the `vcs-speculative-plan-management` attribute, which controls [automatic cancellation of plan-only runs triggered by outdated commits](/terraform/cloud-docs/users-teams-organizations/organizations/vcs-speculative-plan-management).
```

Comment:
@roleesinhaHC @lilincmu I'm not seeing the attribute the org API; am I looking in the wrong place? https://developer.hashicorp.com/terraform/cloud-docs/api-docs/organizations